src/pages/ProfileInfoPage.jsx

```diff
@@ -0,0 +1,208 @@
+          <img src={profileImg} alt="profile image" className="w-20 mr-3" />
+          <div>
+            <h1 className="font-bold">Ahmed Ashraf</h1>
+            <p className="text-gray-500">No risk No Fun</p>
+            <p className="text-gray-500">Jop tittle</p>
+          </div>
+        </div>
+        <div>
+          <p>
+            <FontAwesomeIcon icon={faBookOpen} className={iconStyle} />
+            <span className="font-bold text-blue-600"> 4 Learning</span>
+          </p>
+          <p>
+            <FontAwesomeIcon icon={faArchive} className={iconStyle} />
+            <span className="font-bold text-blue-600"> 2 Achieved</span>
+          </p>
+        </div>
+      </div>
+      <div className="absolute w-full border-b border-gray-400"></div>
+      <form
+        className="grid grid-cols-12 gap-6 px-3 py-10 font-bold leading-8 md:text-xl md:px-5 lg:px-10"
+        onSubmit={handleSubmit}
+      >
+        {/* First Name input */}
+        <input
+          type="text"
+          className={`${inputStyle} col-span-12 md:col-span-6`}
+          placeholder="First Name "
+          value={formState.firstName}
+          onChange={(e) => handleChange(e, "firstName")}
+        />
+
+        {/* Last Name input */}
+        <input
+          type="text"
+          className={`${inputStyle} col-span-12 md:col-span-6`}
+          placeholder="Last Name"
+          value={formState.secondName}
+          onChange={(e) => handleChange(e, "secondName")}
+        />
+
+        {/* Description input */}
+        <input
+          type="text"
+          className={`${inputStyle} col-span-12`}
+          placeholder="Description"
+          value={formState.description}
+          onChange={(e) => handleChange(e, "description")}
+        />
+
+        {/* Job input */}
+        <input
+          type="text"
+          className={`${inputStyle} col-span-12 md:col-span-7 lg:col-span-8`}
+          placeholder="Add your job (Student)"
+          value={formState.jop}
+          onChange={(e) => handleChange(e, "jop")}
+        />
+
+        {/* Add Job button */}
+        <button
+          type="button"
+          className="flex items-center justify-between h-12 col-span-7 text-white bg-black rounded-full md:h-14 s md:col-span-5 lg:col-span-4"
+        >
+          <span className=" text-black bg-white border-gray-400 rounded-full border-[2px] py-[0.4rem] md:py-[0.8rem] mr-3 w-full">
+            + Add{" "}
+          </span>
+          <p className="w-full">a Jop</p>
+        </button>
+      </form>
+
+      {/* Divider */}
+      <div className="absolute w-full border-b border-gray-400"></div>
+
+      {/* Social Networks inputs */}
+      <h1 className="col-span-8 mt-6 mb-3 ml-10 font-bold">Social Networks:</h1>
+      <form
+        className="grid grid-cols-8 gap-6 px-3 py-10 pt-0 font-bold leading-8 md:text-xl md:px-5 lg:px-10"
+        onSubmit={handleSubmit}
+      >
+        {socialNetwork.map((network, index) => (
```

Comment:
Good job, but can you extract these block of code (map) outside of return? so we keep the return lean as soon as we can.